docker/utils.js

```diff
@@ -418,7 +418,7 @@ exports.processBrokenLinks = (
 ) => {
   const __getBadResults = (allUrls) =>
     allUrls
-      .filter((url) => url["StatusCode"] === "404")
+      .filter((url) => url["StatusCode"] !== "200")
```

Comment:
Perhaps we should allow the whole successful 2xx status code range (200-299)

Reply:
Done - I have made an update that allows the whole 2xx range